nuggetsapp/models.py

```diff
@@ -59,5 +66,21 @@ def add_new_nugget_for_user(cls, nugget, user, is_owner=False):
 
     @classmethod
     def get_nuggets_by_user(cls, user):
-        return [x.nugget for x in cls.objects.filter(user = user)]
+        return [x.nugget for x in cls.objects.filter(user = user).filter(is_deleted = False)]
```

Comment:
The double filtering caught me off guard, and found this: http://stackoverflow.com/questions/8164675/chaining-multiple-filter-in-django-is-this-a-bug
Based on the answer, you'll want to do `filter(A, B)` to do the AND query, as opposed to `filter(A).filter(B)` which unintuitively does an OR.  Super lame.